gateway-ha/src/main/java/io/trino/gateway/ha/security/LbOAuthManager.java

```diff
@@ -67,7 +68,18 @@ public class LbOAuthManager
     public LbOAuthManager(OAuthConfiguration configuration, Map<String, String> pagePermissions)
     {
         this.oauthConfig = configuration;
-        this.pagePermissions = pagePermissions;
+        this.pagePermissions = pagePermissions.entrySet().stream()
+                .filter(entry -> entry.getKey() != null || entry.getValue() != null)
+                .collect(toImmutableMap(
+                        entry -> entry.getKey().toUpperCase(),
+                        Map.Entry::getValue));
+    }
+
+    public static Response buildUnauthorizedResponse()
+    {
+        return Response.status(UNAUTHORIZED)
+                .cookie(OidcCookie.delete())
+                .build();
     }
```

Comment:
Formatted code with airlift style changed function location

Reply:
No need to move the method. Please revert.